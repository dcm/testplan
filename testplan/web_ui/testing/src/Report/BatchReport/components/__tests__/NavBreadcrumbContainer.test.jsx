/** @jest-environment jsdom */
// @ts-nocheck
import React from 'react';
import { render } from '@testing-library/react';
import { StyleSheetTestUtils } from 'aphrodite';
import { NavBreadcrumbContainer } from '../';

describe('NavBreadcrumbContainer', () => {

  beforeEach(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterEach(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('renders correctly with dummy child', () => {
    expect(render(
      <NavBreadcrumbContainer>
        <li>Dummy LI element</li>
      </NavBreadcrumbContainer>
    ).container).toMatchSnapshot();
  });

  it('renders correctly without children', () => {
    expect(render(<NavBreadcrumbContainer/>).container).toMatchSnapshot();
  });

});
