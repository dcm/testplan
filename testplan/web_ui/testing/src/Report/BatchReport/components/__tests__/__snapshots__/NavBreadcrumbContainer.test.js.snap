// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBreadcrumbContainer renders correctly with dummy child 1`] = `
<div>
  <div
    class="navBreadcrumbs_koaten"
  >
    <ul
      class="breadcrumbContainer_1qdzaym"
    >
      <li>
        Dummy LI element
      </li>
    </ul>
  </div>
</div>
`;

exports[`NavBreadcrumbContainer renders correctly without children 1`] = `
<div>
  <div
    class="navBreadcrumbs_koaten"
  >
    <ul
      class="breadcrumbContainer_1qdzaym"
    />
  </div>
</div>
`;
